### DISCUSSION TITLE
Query param to decide pagination method

### DISCUSSION THREAD

**Discussion by Hesesses** [spatie/laravel-json-api-paginate#80] [Ideas]
Hello,

Would be nice to be able to use query param to decide between pagination methods, for example:
For cursor pagination: /api/items?pagination=cursor
For page-based pagination: /api/items?pagination=page

Then frontend could decide which method they would like to use.

**Comment by AlexVanderbist** [comment]
I feel like this is a bit out of scope for this package. However, the code in this package is relatively simple to copy paste to your own project and adapt so I'd suggest doing that. Good luck!
